app/assets/javascripts/permits.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
We're never going to be using CoffeeScript in this project, so this file is unnecessary.

Reply:
I will remove it but also exclude `coffee-rails` gem from Gemfile to stop generators from creating CoffeeScript files.